src/transformers/models/owlv2/modeling_owlv2.py

```diff
@@ -1320,6 +1322,7 @@ def normalize_grid_corner_coordinates(self, feature_map: torch.FloatTensor):
         device = feature_map.device
         num_patches = feature_map.shape[1]
 
+        # TODO: Remove numpy usage.
```

Comment:
Any reason this comment is added? I assume this can be deleted since the PR removes numpy usage?